kardinal-cli/cmd/root.go

```diff
@@ -21,6 +22,17 @@ const (
 	devMode              = true
 	kontrolServiceApiUrl = "ad718d90d54d54dd084dea50a9f011af-1140086995.us-east-1.elb.amazonaws.com"
 	kontrolServicePort   = 8080
+
+	kontrolLocationLocalMinikube = "local-minikube"
+	kontrolLocationKloudKontrol  = "kloud-kontrol"
+
+	kontrolClusterResourcesEndpointTmpl = "%s://%s/tenant/%s/cluster-resources"
+
+	localMinikubeKontrolAPIHost = "host.minikube.internal:8080"
+	kloudKontrolAPIHost         = "app.kardinal.dev/api"
```

Comment:
calling this endpoint is failing due to a certificate validation issue. It seems we need to add the `ca-certificates` package inside the container to make it work.

Reply:
adding the config on this commit `8d72b9228a0dba3510c54f0f7d10f34e2f4d2a25` I tested it